### ISSUE TITLE
HTTP Message must be set for OkHttp 3.8.0

### ISSUE THREAD

**Issue by cdeange** [airbnb/okreplay#44]
`OkHttpResponseAdapter` needs to set the message field to ensure that it is compatible with OkHttp 3.8.0 and later.

From [OkHttp's Changelog](https://github.com/square/okhttp/blob/master/CHANGELOG.md):
> The response message is now non-null. This is the "Not Found" in the status line "HTTP 404 Not Found". If you are building responses programmatically (with `new Response.Builder()`) you must now always supply a message. An empty string `""` is permitted. This value was never null on responses returned by OkHttp itself, and it was an old mistake to permit application code to omit a message.

Sample stack trace:
```
java.lang.IllegalStateException: message == null
   at okhttp3.Response$Builder.build(Response.java:431)
   at okreplay.OkHttpResponseAdapter.adapt(OkHttpResponseAdapter.java:20)
   at okreplay.OkReplayInterceptor.intercept(OkReplayInterceptor.java:40)
   at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:92)
   at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain
   ...
```

**Comment by felipecsl** [comment]
Thanks for looking into this and reporting it. It has since been fixed on #40 [here](https://github.com/airbnb/okreplay/pull/40/files#diff-f753ba8e53d2e7b6e1d94228d5ce1b7c)

I'll cut a new release this week with this fix!

**Comment by AlaaOsta** [comment]
There is still an issue when calling an error thru the Retrofit2 Response static method, you need to set the message to empty as well

`  public static <T> Response<T> error(int code, ResponseBody body) {
    if (code < 400) throw new IllegalArgumentException("code < 400: " + code);
    return error(body, new okhttp3.Response.Builder() //
        .code(code)
        .protocol(Protocol.HTTP_1_1)
        **.message("")**
        .request(new Request.Builder().url("http://localhost/").build())
        .build());
  }`
